app/tests/spec/lib/router.js

```diff
@@ -5,16 +5,13 @@
 define(function (require, exports, module) {
   'use strict';
```

Comment:
Not strictly related to this PR, but do you think there is value in asserting that the `routes` object has properties for all of the routes we expect it to? Or does coverage here and elsewhere make that a pointless assertion?